src/serviceorchestration/5_0_0/sd/orchestration/orchestration_sd.tex

```diff
@@ -416,18 +443,23 @@ \subsection{Primitives}
 \begin{tabularx}{\textwidth}{| p{3cm} | X |} \hline
 \rowcolor{gray!33} Type & Description \\ \hline
 \pdef{AddressType}      & Any suitable type chosen by the implementor of service. \\ \hline
+\pdef{CloudID} & A composite string identifier that is intended to be both human and machine-readable. It consists of the cloud name and the organization name, separated by a special delimiter character. Each part must follow PascalCase naming convention. \\ \hline
 \pdef{DateTime}         & Pinpoints a specific moment in time. \\ \hline
 \pdef{ErrorType}        & Any suitable type chosen by the implementor of service. \\ \hline
-\pdef{InterfaceTemplate}& A string identifier of an interface descriptor. \\ \hline
+\pdef{InterfaceName}& A string identifier of an interface descriptor. Must following snake\_case naming convention. \\ \hline
 \pdef{List}$<$A$>$      & An \textit{array} of a known number of items, each having type A. \\ \hline
-\pdef{Name}             & A string identifier that is intended to be both human and machine-readable. \\ \hline
 \pdef{Number}           & Decimal number. \\ \hline
 \pdef{Object}           & Set of primitives and possible further objects. \\ \hline
+\pdef{OperationName}             & A string identifier that is intended to be both human and machine-readable. Must following kebab-case naming convention. \\ \hline
```

Comment:
should be ServiceOperationName here as well. (and in the right place based on the new name)